### PR TITLE
Add early access control plane firewall new payload and structure definition

### PR DESCRIPTION
**Pull request by llDrLove**
Add early access control plane firewall new payload and structure definition. 

Impacts the response of the `List All Kubernetes Clusters`, `Create a New Kubernetes Cluster`, `Update a Kubernetes Cluster` & `Retrieve an Existing Kubernetes Cluster`.

Impacts the payload of `Create a New Kubernetes Cluster` & `Update a Kubernetes Cluster` with the new optional/nullable `control_plane_firewall` property. Both property are also optional/nullable.

### CODE REVIEW

File: specification/resources/kubernetes/models/control_plane_firewall.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  Control plane firewall is in early access. To request access, [contact support](https://cloudsupport.digitalocean.com).
+properties:
+  enable:
+    type: boolean
+    description: Indicates whether the control plane firewall is enabled.
+    example: true
+
+  allowed_addresses:
+    type: array
+    description: An array of addresses (IPv4 or CIDR) allowed to access the control plane.
```
Comment (timoreimann) [inline specification/resources/kubernetes/models/control_plane_firewall.yml:13]:
```suggestion
    description: An array of public addresses (IPv4 or CIDR) allowed to access the control plane.
```

File: specification/resources/kubernetes/models/control_plane_firewall.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    items:
+      type: string
+    example:
+      - 1.2.3.4
+      - 4.3.2.1/32
```
Comment (timoreimann) [inline specification/resources/kubernetes/models/control_plane_firewall.yml:18]: nit: can we reuse the address also used in the response examples?

File: specification/resources/kubernetes/models/control_plane_firewall.yml (hunk 1 above)
Comment (llDrLove) [reply specification/resources/kubernetes/models/control_plane_firewall.yml:13]: Fixed

File: specification/resources/kubernetes/models/control_plane_firewall.yml (hunk 2 above)
Comment (llDrLove) [reply specification/resources/kubernetes/models/control_plane_firewall.yml:18]: Fixed.

File: specification/resources/kubernetes/models/control_plane_firewall.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+type: object
+nullable: true
+description: An object specifying the control plane firewall for the Kubernetes cluster.
+  Control plane firewall is in early access. To request access, [contact support](https://cloudsupport.digitalocean.com).
```
Comment (timoreimann) [inline specification/resources/kubernetes/models/control_plane_firewall.yml:4]:
```suggestion
  Control plane firewall is in early availability. To request access, [contact support](https://cloudsupport.digitalocean.com).
```

Comment (llDrLove) [reply specification/resources/kubernetes/models/control_plane_firewall.yml:4]: Fixed

**Review by danaelhe — APPROVED** [review]
lgtm 👍

**Pull request merged**